### ISSUE TITLE
Extension issue - Can't connect with MySQL

### ISSUE THREAD

**Issue by DoChuan13** [microsoft/azuredatastudio-mysql#210]
- Issue Type: `Bug`
- Extension Name: `azuredatastudio-mysql`
- Extension Version: `0.2.9`
- OS Version: `Darwin arm64 22.4.0`
- VS Code version: `1.43.0`

```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 94,
		"activateCallTime": 0,
		"activateResolvedTime": 2,
		"activationReason": {
			"startup": true,
			"extensionId": {
				"value": "Microsoft.azuredatastudio-mysql",
				"_lower": "microsoft.azuredatastudio-mysql"
			},
			"activationEvent": "*"
		}
	},
	"runtimeErrors": [
		{
			"originalStack": "Error\n    at new ZoneAwareError (/Applications/Azure Data Studio.app/Contents/Resources/app/node_modules.asar/zone.js/dist/zone-error.js:99:37)\n    at S.$onExtensionRuntimeError (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:3536:26641)\n    at y._doInvokeHandler (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2461:13206)\n    at y._invokeHandler (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2461:12890)\n    at y._receiveRequest (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2461:11514)\n    at y._receiveOneMessage (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2461:10192)\n    at vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2461:8301\n    at l.invoke (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:145)\n    at s.deliver (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:2265)\n    at o.fire (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:1843)\n    at f.fire (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:183:19638)\n    at b._receiveMessage (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:183:24219)\n    at vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:183:21753\n    at l.invoke (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:145)\n    at s.deliver (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:2265)\n    at o.fire (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:1843)\n    at o.acceptChunk (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:183:16466)\n    at vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:183:15596\n    at Socket.L (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:4245:13007)\n    at Socket.emit (node:events:526:28)\n    at addChunk (node:internal/streams/readable:315:12)\n    at readableAddChunk (node:internal/streams/readable:289:9)\n    at Readable.push (node:internal/streams/readable:228:10)\n    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)",
			"zoneAwareStack": "Error\n    at S.$onExtensionRuntimeError (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:3536:26641) [<root>]\n    at y._doInvokeHandler (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2461:13206) [<root>]\n    at y._invokeHandler (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2461:12890) [<root>]\n    at y._receiveRequest (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2461:11514) [<root>]\n    at y._receiveOneMessage (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2461:10192) [<root>]\n    at vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2461:8301 [<root>]\n    at l.invoke (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:145) [<root>]\n    at s.deliver (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:2265) [<root>]\n    at o.fire (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:1843) [<root>]\n    at f.fire (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:183:19638) [<root>]\n    at b._receiveMessage (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:183:24219) [<root>]\n    at vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:183:21753 [<root>]\n    at l.invoke (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:145) [<root>]\n    at s.deliver (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:2265) [<root>]\n    at o.fire (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:136:1843) [<root>]\n    at o.acceptChunk (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:183:16466) [<root>]\n    at vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:183:15596 [<root>]\n    at Socket.L (vscode-file://vscode-app/Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:4245:13007) [<root>]\n    at Socket.emit (node:events:526:28) [<root>]\n    at addChunk (node:internal/streams/readable:315:12) [<root>]\n    at readableAddChunk (node:internal/streams/readable:289:9) [<root>]\n    at Readable.push (node:internal/streams/readable:228:10) [<root>]\n    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23) [<root>]",
			"name": "TypeError",
			"message": "The \"path\" argument must be of type string. Received null"
		}
	],
	"runningLocation": {
		"affinity": 0,
		"kind": 1
	}
}
```

**Comment by nitishsharma25** [comment]
@DoChuan13 currently, 0.2.8 is the latest stable version of our extension published in Azure Data Studio, could you use this and check if you are facing this issue?

Also, we have not yet released 0.2.9.

**Comment by nitishsharma25** [comment]
Closing the issue since there is no response and it is not relevant.
